Makefile

```diff
@@ -19,7 +19,7 @@ GOFMT := gofmt
 
 .DEFAULT_GOAL=all
 
-all: vet lint build fmt
+all: clean vet lint build fmt
```

Comment:
you may not want to do a clean every time you build, right?
doing make clean && make all could be a better way around this one. What do you think?

Reply:
I was going for a default clean to happen every time. But `make clean && make all` works too. Removed it.